Services/CO.CDP.Organisation.WebApi/Api/Organisation.cs

```diff
@@ -645,6 +645,85 @@ await useCase.Execute((organisationId, personInviteId))
 
         return app;
     }
+
+    public static RouteGroupBuilder UseManageApiKeyEndpoints(this RouteGroupBuilder app)
+    {
+        app.MapGet("/{organisationId}/api-keys",
+            async (Guid organisationId, IUseCase<Guid, IEnumerable<Model.AuthenticationKey>> useCase) =>
+               await useCase.Execute(organisationId)
+                   .AndThen(entities => entities != null ? Results.Ok(entities) : Results.NotFound()))
```

Comment:
Can `entities` actually be `null`, or just an empty collection?

Reply:
It will be just empty collection

Reply:
So why is there a null check here?

Reply:
removed null check